blockstack_client/backend/drivers/ipfs.py

```diff
@@ -207,25 +199,33 @@ def handles_url( url ):
     matches what your driver does.  Another common strategy
     is to check if the URL matches a particular regex.
     """
-    pattern = re.compile("Qm[1-9A-HJ-NP-Za-km-z]{44}")
-    if pattern.match(url):
+    if url.startswith("/ipfs/") or url.startswith("/ipns/"):
       return True
     else:
+      # if it starts with a valid CID: https://github.com/ipld/cid
+      #   return True
+      # else
       return False
 
 
 def make_mutable_url( data_id ):
     """
     Get data by URL
     """
-    api = ipfsapi.connect(IPFS_SERVER, IPFS_PORT)
-
     key = 'mutable-{}'.format(data_id.replace( "/", r"\x2f" ))
 
     ipfs_key_gen( key )
    
-    r = api.name_publish( hash_data(''), opts = {'key':key})
-    return r['Name']
+    url = None
+
+    try:
+      r = ipfs_api.name_publish( hash_data(''), opts = {'key':key})
```

Comment:
Every user will have a different public key, which means the 'name' being published is different, i think this code is just setting the 'default' value when starting up